doc/rtd/explanation/format.rst

```diff
@@ -24,7 +24,7 @@ These things include:
 - *and many more...*
 
 .. note::
-   This file must be valid YAML syntax.
+   This file must be valid `YAML 1.1`_ syntax.
```

Comment:
I know you didn't write this line, but while we're touching this line can we please update it to something that makes a little more sense?

The file isn't a syntax. YAML isn't a syntax either. YAML is a standardized language for defining data objects, and it has a syntax for describing these data objects as a stream of characters. Cloud-config is a format which uses the YAML syntax described in specification version 1.1 and a set of keys defined [in the jsonschema](https://github.com/canonical/cloud-init/blob/main/cloudinit/config/schemas/schema-cloud-config-v1.json).

> This file ...

Additionally, this configuration is often times not even a file. Users may copy-paste a cloud-config into a web interface, for example, and ultimately on the instance they usually do not write a configuration file, so the mention of it being a file may be misleading to some.

Perhaps this paragraph:

> Cloud-config is the simplest way to accomplish some things via user data.
> Using cloud-config syntax, the user can specify certain things in a
> human-friendly format.

Could be updated to say something like:

> Cloud-config is the preferred user data format. The cloud config format is a 
> declarative syntax which uses YAML version 1.1 with [keys](https://github.com/canonical/cloud-init/blob/main/cloudinit/config/schemas/schema-cloud-config-v1.json) that describe instance
> state. Cloud-config can be used to define how an instance should be configured in
> a human-friendly format.

And then remove the `This file must be ...` note?